web/src/Web.App/Services/SuggestService.cs

```diff
@@ -94,16 +73,84 @@ public async Task<IEnumerable<SuggestValue<LocalAuthority>>> LocalAuthoritySugge
         }
 
         var suggestions = await establishmentApi.SuggestLocalAuthorities(search, query).GetResultOrThrow<SuggestOutput<LocalAuthority>>();
-        return suggestions.Results.Select(value =>
+        return suggestions.Results.Select(LocalAuthoritySuggestValue);
+    }
+
+    private static SuggestValue<LocalAuthority> LocalAuthoritySuggestValue(SuggestValue<LocalAuthority> value)
+    {
+        var text = value.Text?.Replace("*", "");
+
+        if (text != value.Document?.Name)
         {
-            var text = value.Text?.Replace("*", "");
+            value.Text = value.Document?.Name;
+        }
 
-            if (text != value.Document?.Name)
-            {
-                value.Text = value.Document?.Name;
-            }
+        return value;
+    }
+
+
+    private static SuggestValue<School> SchoolSuggestValue(SuggestValue<School> value)
```

Comment:
chore: Some unit test coverage to assert the various match combinations would be great